test/e2e/suite/nodeadm_test.go

```diff
@@ -19,6 +19,7 @@ import (
 	s3v2 "github.com/aws/aws-sdk-go-v2/service/s3"
 	ssmv2 "github.com/aws/aws-sdk-go-v2/service/ssm"
 	"github.com/go-logr/logr"
+	"github.com/onsi/ginkgo/v2"
```

Comment:
we already have the ginkgo package imported
just use `By` instead of `ginkgo.By`